### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by manuelroth**
This is the first release of the scroll graphic tool. See [the readme](https://github.com/nzzdev/Q-scroll-graphic#functionality) for in introduction to the functionality of the tool.

This branch is deployed on staging: https://q.st-staging.nzz.ch/item/scroll_graphic-0. Its easiest to test it directly in an article: https://www-stage.nzz.ch/editorial-tech-grafikabfolge-ld.1334926

### CODE REVIEW

File: views/components/Scroller.svelte [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+          if (entry.isIntersecting) {
+            if (index === -1) handlers.push(update);
+          } else {
+            update();
+            if (index !== -1) handlers.splice(index, 1);
+          }
+        });
+      },
+      {
+        rootMargin: "400px 0px", // TODO why 400?
```
Comment (philipkueng) [inline views/components/Scroller.svelte:29]: Remove or elaborate comment

**Review by philipkueng — APPROVED** [review]
Beside the two things mentioned, everything looks fine to me. Congratz on another amazing tool Manuel! Well done!

File: views/components/Scroller.svelte (hunk 1 above)
Comment (manuelroth) [reply views/components/Scroller.svelte]: I removed the comment in https://github.com/nzzdev/Q-scroll-graphic/pull/1/commits/75e61359f0be5faaf9854c4dcf58b288c76381a9. This is actually coming from the source repository [svelte-scroller](https://github.com/sveltejs/svelte-scroller)

File: views/components/Footer.svelte [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+                rel="noopener noreferrer"
+              >
+                {source.text}
+              </a>
+            {:else}
+              {source.text}
+            {/if}
+            {#if index !== item.sources.length - 1 && item.sources[index + 1] !== ""}
+              ,&nbsp;
+            {/if}
```
[the remark this answers is not in the document]
Comment (manuelroth) [reply views/components/Footer.svelte]: I have adjusted the template and it works now also with the formatting:
Screenshot 2021-07-29 at 15 26 24

**Pull request merged**